sbpy/spectroscopy/tests/data/HCN.csv

```diff
@@ -0,0 +1,29 @@
+UT_Date,Time,T_B,log(Q),error_top,error_bottom,Q,Q_top,Q_bottom

```

Comment:
I'm very confused about this error since I don't have neither a module nor a script under the name prodrate_simple, I searched under all my scripts in spectroscopy and none of these contained the neither a module named prodrate_simple (I'm pretty sure I used to though when I still didn't have a name for prodrate_np function) nor a line that said hcn = Table.read('HCN.csv' ....) I purposely have the data_dir function that deals with the paths of the data so I use "Table.read(data_dir('HCN.csv' ...)",  (as seen under test_spec_remote.py). Could it be you have one of the older versions instead of my new version from this PR? If not let me know so that I can look at the root of this error more in depth, but as of now I'm inclined to think it's not the same version as the one I just committed to the PR.

Reply:
They do pass for me, I'll take a look at the CI failures now

Reply:
@migueldvb The CI failure seems to be an import error with astroquery.jplspec, pretty sure it might be because the new version of astroquery isn't out just yet.

Reply:
That's a good point. I think that adding astroquery's master branch as a dependency in the travis configuration file should fix that. `https://github.com/astropy/astroquery/archive/master.zip` should work. I think that it is OK to change this as part of this pull request to make sure that CI builds pass.

Reply:
Perhaps merging #51 first and rebasing your pull request should fix the problem. Let me try that and will let you know.